server/models/article.js

```diff
@@ -38,6 +57,42 @@ articleSchema.statics.urisAreEqual = (uri1, uri2) => {
   return normalizeURI(uri1) === normalizeURI(uri2);
 };
 
+articleSchema.methods.getMercuryInfo = function getMercuryInfo() {
+  const encodedURI = encodeURIComponent(this.uri);
```

Comment:
Yeah, encodeURIComponent is a core JS function that escapes characters like / and ? that have special meanings in URIs, so that you can embed things within URIs without them doing anything weird. So I think it's good to call it on any query parameters when constructing a URI.